exercises/pascals-triangle/description.md

```diff
@@ -4,10 +4,11 @@ In Pascal's Triangle each number is computed by adding the numbers to
 the right and left of the current position in the previous row.
 
 ```text
-    1
-   1 1
-  1 2 1
- 1 3 3 1
-1 4 6 4 1
+      1
+     1 1
+    1 2 1
+   1 3 3 1
+  1 4 6 4 1
+1 5 10 10 5 1
```

Comment:
Why do you want to add this row?

I think the first 5 rows demonstrate the triangle sufficently, and adding 2 digit numbers makes the formatting difficult. The current formatting is not correctly triangular.

Reply:
Sorry for not replying on your comment.
Ah i wrote this line for me because i wanted simply more example data and then thought it would make no big difference because somehow the alignment is nearly symmetric.
@Insti So you think it is better to leave it unchanged?

Reply:
Yes, unchanged is better.